src/input_xml.F90

```diff
@@ -1049,8 +1049,9 @@ subroutine read_geometry_xml()
     do i = 1, n_cells
       c => cells(i)
 
-      ! Initialize the number of cell instances - this is a base case for distribcells
+      ! Initialize distribcell instances and distribcell index
```

Comment:
`instances` is a count of how many times a particular distributed cell appears in the geometry.  For example `instances = 4` for a pin cell in a 2x2 lattice.  `distribcell_ind` will range from 1 to the number of distributed cells.  For example, test_filter_distribcell/case-3 has tallies on cells 1, 60, and 27 so `distribcell_ind` might equal 1, 2, or 3.